File: src/main/java/com/EmosewaPixel/pixellib/miscUtils/ItemUtils.java
package com.EmosewaPixel.pixellib.miscutils;

import net.minecraft.entity.item.ItemEntity;
import net.minecraft.item.ItemStack;
import net.minecraft.util.math.BlockPos;
import net.minecraft.world.World;

//This class contains useful functions for working with items
public class ItemUtils {
    public static void spawnItemInWorld(World world, BlockPos pos, ItemStack stack) {
        world.addEntity(new ItemEntity(world, pos.getX() + 0.5F,
                pos.getY() + 0.5F, pos.getZ() + 0.5F, stack));
    }
}

File: src/main/java/com/EmosewaPixel/pixellib/miscUtils/ItemUtils.kt
package com.EmosewaPixel.pixellib.miscutils

import net.minecraft.entity.item.ItemEntity
import net.minecraft.item.ItemStack
import net.minecraft.util.math.BlockPos
import net.minecraft.world.World

//This class contains useful functions for working with items
object ItemUtils {
    @JvmStatic
    fun spawnItemInWorld(world: World, pos: BlockPos, stack: ItemStack) {
        world.addEntity(ItemEntity(world, (pos.x + 0.5f).toDouble(),
                (pos.y + 0.5f).toDouble(), (pos.z + 0.5f).toDouble(), stack))
    }
}